### PR TITLE
[DDW-317] Handle random stake pool data during the first shelley epochs

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR handled random stake pool data during the first shelley epochs by using sandclock svg icon and greying out color band.
[Jira Ticket](https://jira.iohk.io/browse/DDW-317)

## Todos
- [x] Update stakepools list page to show sandclock svg icon for ranking and greyed out color band during the first shelley epochs

### English
Coming...
### Japanese
Coming...
## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1597228607040600)

### App
- [x] Go to `Staking` => `Stake Pools` and make sure stake pools list is shown as above scenario during the first shelley epochs

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by a-rukin** [comment]
image

I see two issues here: 
- it should be right alligned 
- sandclock is not vertically centered

**Comment by a-rukin** [comment]
image
is this newely broken or and old one?

File: source/renderer/app/config/epochsConfig.js [hunk 1]
```diff
@@ -11,15 +11,23 @@ export const START_TIME_DEVELOPMENT = 1541808003;
...
 export const SLOT_DURATION_STN = 1; // unit: seconds
 export const SLOT_DURATION_ITN = 2; // unit: seconds
+export const SLOT_DURATION_SHELLEY = isDev ? 7 : 20;
 
 export const TOTAL_SLOTS = 432000;
 export const TOTAL_SLOTS_STN = 21600;
 export const TOTAL_SLOTS_ITN =
   isDev || isTest || isIncentivizedTestnetSelfnode ? 150 : 43200;
+// eslint-disable-next-line
+export const TOTAL_SLOTS_SHELLEY = isDev
```
Comment (nikolaglumac) [inline source/renderer/app/config/epochsConfig.js:21]: @yakovkaravelov please remove this.

File: source/renderer/app/config/epochsConfig.js [hunk 2]
```diff
@@ -11,15 +11,23 @@ export const START_TIME_DEVELOPMENT = 1541808003;
...
+  ? 1541808003
+  : isTest
+  ? 1537941600
+  : 1506203091;
 
 export const EPOCH_LENGTH = SLOT_DURATION * TOTAL_SLOTS; // 5 days | unit: seconds
 export const EPOCH_LENGTH_STN = SLOT_DURATION_STN * TOTAL_SLOTS_STN; // 6 hours | unit: seconds
 export const EPOCH_LENGTH_ITN = SLOT_DURATION_ITN * TOTAL_SLOTS_ITN; // 1 day / 5 minutes (isDev || isTest || isIncentivizedTestnetSelfnode = true) | unit: seconds
+export const EPOCH_LENGTH_SHELLEY = SLOT_DURATION_SHELLEY * TOTAL_SLOTS_SHELLEY;
```
Comment (nikolaglumac) [inline source/renderer/app/config/epochsConfig.js:30]: @yakovkaravelov please remove this.

File: source/renderer/app/stores/NetworkStatusStore.js [hunk 3]
```diff
@@ -120,6 +121,7 @@ export default class NetworkStatusStore extends Store {
   @observable isTlsCertInvalid: boolean = false;
   @observable stateDirectoryPath: string = '';
   @observable isShelleyActivated: boolean = false;
+  @observable isShelleyDataAvailable: boolean = false;
```
Comment (nikolaglumac) [inline source/renderer/app/stores/NetworkStatusStore.js:124]: @yakovkaravelov please remove this.

File: source/renderer/app/stores/NetworkStatusStore.js [hunk 4]
```diff
@@ -592,20 +594,27 @@ export default class NetworkStatusStore extends Store {
...
+        isShelleyPending,
+      } = this;
       const { decentralizationLevel, hardforkAt } = networkParameters;
       const epochStartTime = get(hardforkAt, 'epoch_start_time', '');
 
       if (hardforkAt) {
         const currentTimeStamp = new Date().getTime();
         const hardforkStartTime = new Date(epochStartTime).getTime();
         isShelleyActivated = currentTimeStamp >= hardforkStartTime;
+        isShelleyDataAvailable =
```
Comment (nikolaglumac) [inline source/renderer/app/stores/NetworkStatusStore.js:609]: @yakovkaravelov please remove this.

**Pull request merged**